package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "remix-auth",
-  "version": "3.5.1",
+  "version": "3.6.0",
```

This is changed automatically with a GitHub Action
```suggestion
  "version": "3.5.1",
```

---

Done!  Thanks for the feedback